From 82d43b95791c5303f6e83f9de496aa377cb56702 Mon Sep 17 00:00:00 2001
From: Jan Luebbe <sho@stratum0.net>
Date: Mon, 13 Jun 2022 16:40:18 +0200
Subject: [PATCH] hwdb: analyzers: remove generic "STM Device in DFU Mode"

The USB ID v0483pDF11 is used by the ROM code in many STMicroelectronics
devices (for firmware download) and not just signal analyzers.

(cherry picked from commit 5d049ff9204b9aad48c62c296def4daa4b53005e)

Related: #2087778
---
 hwdb.d/70-analyzers.hwdb | 1 -
 1 file changed, 1 deletion(-)

diff --git a/hwdb.d/70-analyzers.hwdb b/hwdb.d/70-analyzers.hwdb
index 899ece3a01..0a1911507a 100644
--- a/hwdb.d/70-analyzers.hwdb
+++ b/hwdb.d/70-analyzers.hwdb
@@ -29,7 +29,6 @@ usb:v1679p3001*
 
 # Power Delivery Analyzers
 usb:v1679p6003*
-usb:v0483pDF11*
  ID_SIGNAL_ANALYZER=1
 
 ###########################################################
